fedimint-testing/src/gateway.rs

```diff
@@ -84,9 +84,32 @@ impl GatewayTest {
             .get_rpc()
             .await
             .with_password(Some(DEFAULT_GATEWAY_PASSWORD.to_string()));
-        rpc.connect_federation(ConnectFedPayload { invite_code })
+        rpc.connect_federation(ConnectFedPayload {
+            invite_code,
+            routing_fees: None,
+        })
+        .await
+        .unwrap()
+    }
+
+    /// Connects to a new federation passing a routing fees, and stores the info
+    pub async fn connect_fed_with_routing_fees(
```

Comment:
I don't think this function is necessary, lets just keep `connect_fed` as is and allow them to change the fees later